web/gds-user-ui/src/locales/en/messages.po

```diff
@@ -759,7 +786,7 @@ msgstr "Name, Address, Country, National Identifier"
 msgid "National Identification"
 msgstr "National Identification"
 
-#: src/components/OtherJuridictions/index.tsx:40
+#: src/components/OtherJurisdictions/index.tsx:40
```

Comment:
See my note above - unfortunately we'll have to live with the filepath typo for now. Would you mind changing it back?